docs/components/popovers.md

```diff
@@ -181,7 +181,7 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
     </tr>
     <tr>
       <td>container</td>
-      <td>string | false</td>
+      <td>string | element | boolean</td>
```

Comment:
If you felt like it, you could also change `false` to `boolean` on the tooltip container too:

From:
```
<td>string | element | false</td>
```
to:
```
<td>string | element | boolean</td>
```